lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -14,8 +14,26 @@ class BulkDataPatientExportSequence < SequenceBase
       requires :token
       conformance_supports :Patient
 
-      def check_export_kick_off(klass)
-        reply = export_kick_off(klass)
+      attr_writer :run_all_kick_off_tests
+
+      def check_export_kick_off(klass, search_params: nil)
+        @search_params = search_params
+
+        reply = export_kick_off(klass, search_params: search_params)
+
+        # Servers unable to support _type SHOULD return an error and OperationOutcome resource
+        # so clients can re-submit a request omitting the _type parameter.
+        if search_params.present? && reply.code > 400
+          response_body = JSON.parse(reply.body)
+          message = ''
+          response_body['issue'].each do |issue|
+            message += issue['diagnostics'].presence || ''
+          end
+
+          skip message
+        end
+
+        @server_support_type_parameter = search_params.present?
```

Comment:
How does the presence of any search param indicate that the server supports the _type parameter? Line 26 also seems to be assuming that `_type` is the only possible search param.

Reply:
It is true that _type is the only parameter for now. Other parameters will be add later. Still worth to fix it.